TheMealDB/TheMealDB/API/Model/Meal.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  Meal.swift
+//  TheMealDB
+//
+//  Created by mac on 12/18/18.
+//  Copyright © 2018 mac. All rights reserved.
+//
+
+import Foundation
+import ObjectMapper
+
+class Meal : Mappable {
+    var idMeal = ""
+    var strMeal = ""
+    var strArea = ""
+    var strCategory = ""
+    var strInstructions = ""
+    var strMealThumb = ""
+    var resource = [resources]()
+    
+    required init(map: Map) {
+        mapping(map: map)
+    }
+    
+    func mapping(map: Map) {
+        idMeal <- map["idMeal"]
+        strMeal <- map["strMeal"]
+        strArea <- map["strArea"]
+        strCategory <- map["strCategory"]
+        strInstructions <- map["strInstructions"]
+        strMealThumb <- map["strMealThumb"]
+        (1...20).forEach { index in
+            var ingredient = ""
+            var measure = ""
+            ingredient <- map["strIngredient\(index)"]
+            measure <- map["strMeasure\(index)"]
+            if ingredient != "" {
+                resource.append(resources(ingredient: ingredient, measure: measure))
+            }
+        }
+    }
+}
+
+struct resources {
```

Comment:
```suggestion
struct Resource {
```
Tên class và struct là viết thường hả em?